cloverage/src/cloverage/coverage.clj

```diff
@@ -269,6 +269,10 @@
             (mark-loaded namespace)))
 
         (println "Instrumented namespaces.")
+        ;; load runner multimethod definition from other dependencies
+        (when-not (#{:clojure.test :midje} runner)
+          (try (require (symbol (format "%s.cloverage.clj" (name runner))))
```

Comment:
Wait; why is there a .clj in the required sym? How does one use this?

Reply:
I'm sorry; I should have waited till after I got a chance to test this with my external lib before sending a ping.

Reply:
OK, this time I've tried it with my branch of `circleci.test` and it seems to do the trick. Would be great if #195 could be merged too.

Thanks!